cumulusci/tasks/salesforce/Deploy.py

```diff
@@ -55,6 +56,9 @@ class Deploy(BaseSalesforceMetadataApiTask):
         "transforms": {
             "description": "Apply source transforms before deploying. See the CumulusCI documentation for details on how to specify transforms."
         },
+        "rest_deploy": {
+            "description": "If True, deploy metadata with Apex Testing using REST API"
```

Comment:
Apex tests don't _necessarily_ run on a deploy that doesn't include Apex.

```suggestion
            "description": "If True, deploy metadata using REST API"
```